src/Tickets/Commerce/Payments_Tab.php

```diff
@@ -34,6 +37,43 @@ class Payments_Tab extends tad_DI52_ServiceProvider {
 	 */
 	public static $option_page_created_meta_key = 'tec_tc_payments_page_created';
 
+	/**
+	 * Meta key for page creation flag.
+	 *
+	 * @since 5.2.1
```

Comment:
Looks like we're introducing this var, I think we should set the `@since TBD`

Reply:
Good catch. Copy/paste error